app/models/ability.rb

```diff
@@ -4,9 +4,15 @@ class Ability
   def initialize(user)
     user ||= User.new
 
-    can [:read, :edit, :update, :destroy, :download, :deploy, :activity], App,
-        app_members: { user_id: user.id }
+    can [
+           :read, :edit, :update, :destroy,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.